setup.py

```diff
@@ -83,8 +83,11 @@ def yellow(text) : return text
 # -----------------------------------------------------------------------------
 # Classes and functions
 # -----------------------------------------------------------------------------
-
-copy("LICENSE.txt", "bokeh/")
+try:
+    copy("LICENSE.txt", "bokeh/")
+except FileNotFoundError:
+    # setup.py called from bokehjs dir by gulp watch process
+    copy("../LICENSE.txt", "../bokeh/")
```

Comment:
I can see that this would work, but I wonder if there isn't a better way to accomplish this. If not, there at least needs to be a comment explaining what's going on.